packages/editor/src/components/post-title/index.js

```diff
@@ -169,7 +170,7 @@ function PostTitle( _, forwardedRef ) {
 				( firstBlock.name === 'core/heading' ||
 					firstBlock.name === 'core/paragraph' )
 			) {
-				onUpdate( firstBlock.attributes.content );
+				onUpdate( stripHTML( firstBlock.attributes.content ) );
```

Comment:
What is the `plaintext` for if we need to stripHTML here?

Reply:
`plainText` is a non-block representation of the content whereas `content` may be an array of blocks.

So in the example provided in the description...

`content` is:

```js
[
    {
        "clientId": "ac523474-8aa7-4fa8-b732-2863e95f03d5",
        "name": "core/paragraph",
        "isValid": true,
        "attributes": {
            "content": "<strong>Puerto Rican Power: Pedraza Dominates Molina</strong><strong></strong>",
            "dropCap": false
        },
        "innerBlocks": []
    },
    {
        "clientId": "49782d58-9f9d-4617-bcfb-7ce93a6a34cb",
        "name": "core/paragraph",
        "isValid": true,
        "attributes": {
            "content": "<strong>Ajagba Decisions Rice in Heavyweight Co-Feature</strong>",
            "dropCap": false
        },
        "innerBlocks": []
    }
]
```

`plainText` is:

```
Puerto Rican Power: Pedraza Dominates Molina

Ajagba Decisions Rice in Heavyweight Co-Feature
```

That's why this PR is stripping HTML from the `content`.

Reply:
I should add that we can't just use `plainText` here because the title seems to be designed to break into new blocks within the post-content. So you'll notice that when pasted `Ajagba Decisions Rice in Heavyweight Co-Feature` becomes a block within the post content and not part of the post title.